### ISSUE TITLE
Recent event doesn't appear when setting a high ranking score in lazer

### ISSUE THREAD

**Issue by cl8n** [ppy/osu-web#11188]
(this thing) 
![image](https://github.com/ppy/osu-web/assets/9954349/18a695d6-8cab-4da5-ad4e-e84c12a1accb)

opening the issue here, but I'm not sure if this is supposed to be the responsibility of osu-web or the score queue.

**Comment by nanaya** [comment]
The calculation itself is somewhere else but the event row should probably be created by osu-web through interop endpoint? 🤔

**Comment by cl8n** [comment]
makes sense. I won't be implementing this but some more random thoughts on briefly looking into it:

- this is directly related to the "lost first place" event, which I think should be re-implemented at the same time
  - and also that one has some special behavior to be deleted if the user who caused it turns out to be a cheater, no idea how that works though.
- would be cool if the stable and lazer version of these events could be distinguished, and filtered for display based on the viewer's "lazer mode" setting
  - if this is done, then setting a stable score would actually need to create one of each event I think? because the rankings won't be the same

**Comment by NeofetchNpc** [comment]
when is it implemented?

**Comment by peppy** [comment]
> when is it implemented?

I can understand you are excited to see this happen, but please know that we are doing our best to get everything working how everyone wants it as fast as we can. There's no ETA yet but it's on my high priority list.
